src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -9,6 +9,8 @@
     <UsingMSBuildSdkSqlProj>true</UsingMSBuildSdkSqlProj>
     <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)' != 'Current'">$(MSBuildAllProjects);$(MsBuildThisFileFullPath)</MSBuildAllProjects>
     <SqlServerVersion>Sql150</SqlServerVersion>
+    <TreatTSqlWarningsAsErrors>False</TreatTSqlWarningsAsErrors>
```

Comment:
I'm wondering if we even need to default these properties here at all. You already have to explicitly set `TreatTSqlWarningsAsErrors` to `True` in order to enable the feature, so the default is already `False` semantically. So I think we don't even need to make changes to this file.